views/account/settings.hbs

```diff
@@ -76,6 +76,7 @@
                 {{> "lib/forms/password-hint" plaintext=false style="display: inline;"}}
             </span>
         </div>
+<!-- the following is hidden until the feature is fixed
```

Comment:
You should use `{{#if false}}` to comment out this section, cause HTML comments like here are still sender to the client but are hidden there and never used. This saves the user a couple of bytes and make him happy. 🙃